x/leverage/keeper/keeper_test.go

```diff
@@ -28,6 +28,9 @@ func (s *IntegrationTestSuite) TestSupply() {
 	s.borrow(borrower, coin(atomDenom, 10_000000))
 	s.tk.SetBorrow(ctx, borrower, coin(atomDenom, 60_000000))
 
+	// create a supplier that will exceed token's default MaxSupply
+	whale := s.newAccount(coin(umeeDenom, 1000000_000000))
```

Comment:
```suggestion
	whale := s.newAccount(coin(umeeDenom, 1_000_000_000000))
```

Reply:
or... let's define and export in fixtures a `DefaultMaxSupply=100_000000` and here let's use `fixtures.DefaultMaxSupply*10`.